### PR TITLE
Add an "Edit Versions" listing to the Admin menu

### PR DESCRIPTION
**Pull request by ericholscher**
This will let users easily find the current version editing UI,
and be the place we'll eventually move it after a refactor.

### CODE REVIEW

File: readthedocs/templates/projects/project_edit_base.html [hunk 1]
```diff
@@ -28,6 +28,8 @@
         {% if USE_PROMOS %}
           <li class="{% block project-ads-active %}{% endblock %}"><a href="{% url "projects_advertising" project.slug %}">{% trans "Advertising" %} </a></li>
         {% endif %}
+        <!-- Add a link to the dashboard versions page so users can find it -->
+        <li><a href="{% url "project_version_list" project.slug %}">{% trans "Edit Versions" %}</a></li>
```
[the remark this answers is not in the document]
Comment (saadmk11) [reply readthedocs/templates/projects/project_edit_base.html]: Can't we use font awesome icon? https://fontawesome.com/icons/external-link-alt?style=solid
`<i class="fas fa-external-link-alt"></i>`

[the remark this answers is not in the document]
Comment (saadmk11) [reply readthedocs/templates/projects/project_edit_base.html]: @ericholscher We can use unicode value of the icon. like we use here. `\f08e`
https://github.com/readthedocs/readthedocs.org/blob/b9aff0ce413fe5d4ffaabc2f98b7fb89019dae7f/media/css/core.css#L821-L824

[the remark this answers is not in the document]
Comment (agjohnson) [reply readthedocs/templates/projects/project_edit_base.html]: We do in fact use font awesome on the community site, however I don't think we want that icon on this link. The icon communicates that the user will be taken away from our site, when we're really just duplicating the link to the misplaced version page -- the user likely won't notice the difference.

[the remark this answers is not in the document]
Comment (agjohnson) [reply readthedocs/templates/projects/project_edit_base.html]: It's sort of odd to put the versions at the bottom of the nav list though, I'd say bump it up to where it was before and where we'll want the version admin in that nav list order when we revert the change that removed the admin dashboard page. It's less findable at the bottom of the list.

**Comment by agjohnson** [comment]
To clarify, we had some conversations around this: we removed the wrong UI. Or rather, we should have moved the dashboard version list page into the admin dashboard, replacing that mess of a UI. I might have originally brought up the idea to remove the dashboard version page, but I might not have been clear that the project dashboard version list page is the one that is unnecessary.

We already have a version list on the project dashboard page, the version listing page therefore is redundant for reader users, and the admin interactions that a maintainer takes shouldn't be on the project dashboard, but should be on the admin dashboard instead.

So the ultimate plan is to backtrack a little on this change, move the existing UI into the admin dashboard, and remove the dashboard version listing page. We'll likely have to do the reverse linking then and temporarily link the project dashboard version button to the admin dashboard version listing view.

**Comment by humitos** [comment]
> So the ultimate plan is to backtrack a little on this change, move the existing UI into the admin dashboard, and remove the dashboard version listing page.

Removing the dashboard listing page, will remove this page https://readthedocs.org/projects/sphinx-hoverxref/versions/ that is accessible for anonymous users. In that page, you can see all the active versions and their docs. Although, the same behavior is at the project home page as well: https://readthedocs.org/projects/sphinx-hoverxref/

**Comment by agjohnson** [comment]
> Although, the same behavior is at the project home page as well

Yup, that's my point above. The project dashboard version list view is not a helpful view for reader users and can be removed entirely. Leo was already instructed to make this change in the project spec, but we removed the wrong UI.

**Pull request merged**